src/gui/optionsdialog.ui

```diff
@@ -641,17 +662,17 @@
                    </property>
                    <item>
                     <property name="text">
-                     <string comment="Delete backup logs older than 10 months">days</string>
+                     <string>days</string>
```

Comment:
I think these can be added back too. Also the "months" should be corrected to "days".

Reply:
Added all of them back.